_pages/about-us/teams/oa.md

```diff
@@ -35,6 +35,16 @@ If you are within TTS and your procurement need costs less than the micro-purcha
 If you are within TTS and your procurement need costs more than the micro-purchase threshold ($10,000), please visit the 
 [Procurements over $10,000]({{site.baseurl}}/procurements-over-10000) section in the handbook.
 
+### TTS Assisted Acquisition
+
+Procurement of services and/or products over $10k that will partially or fully support your partner(s) outside of TTS (including other GSA offices and partner agencies), paid partially or fully by your partner(s), will be considered an assisted acquisition. 
+
+However, OA is currently set up as a shared service, so please keep in mind that we cannot work directly with outside partners. In order for us to do an assisted acquisition, the request must come through one of the following TTS Business Units (BU). Also note that each BU has its own assisted acquisition process as well as vetting criteria. Please contact the following BUs for more information and/or to get started:
```

Comment:
This paragraph sounds like it's written for an outside partner, but the links below are mostly to internal-only docs. Are you envisioning this content being for TTS folks partnering with OA on Assisted Acq projects, or for people outside TTS?

Reply:
The idea is that internal clients will work with us to help their outside partners with acquisitions. So the content would be for TTS folks partnering with OA on Assisted Acq projects.

Reply:
Gotcha! I think it's a little odd to suggest folks contact a business unit, if you're already working with the TTS business units to start. Mind if I edit a bit for clarity?

Reply:
Let me run it by Esther. I didn't write this, just submitted the pull request, so need her approval first.

Reply:
Per Esther, the POC info is for other offices (not part of 18F, CoE, OPP) to contact. Would like to leave as is.

Reply:
Not sure how to resolve and i cant see who is listed as the POC but anyone at TTS interested  in 18F conducting  an assisted acquisition should reach out in  #18f-bd to ensure it is vetted though our established  business development process. External partners use  inquiries18F@gsa.gov.